README.md

```diff
@@ -17,10 +17,17 @@ nix develop #on a large shared machine you might want to run nix --cores=<some n
 the first time you run this it will take a very long time, because it will build most of the software universe from
 scratch for you, including pytorch and tensorflow, let it run overnight. But after that all subsequent calls
 to nix develop will drop you into your shiny new environment. If you're ready to package up your environment into
-a container you can run
+a container you can run:
 
 ```sh
-nix build
+nix build .#container
 ```
 
 this takes a bit of time, so maybe start it before your next meeting.
+
```

Comment:
if you're following the steps in the readme you do, because you've generated the flake from the built in template. `nix build` should be run every time you `nix shell` to ensure any changes to the build are reflected in the gcroot. I'm going to leave this as is for now, this will get rewritten as we move toward a more ideal incarnation of this idea